bindings/rust/bench/src/rustls.rs

```diff
@@ -48,21 +54,14 @@ impl RustlsHarness {
         ))
     }
 
-    /// Read all incoming data, process it, and write it out
-    /// Works for handshaking and data transfer
-    fn process_conn(&mut self, mode: Mode) -> Result<(), std::io::Error> {
-        match match mode {
-            Mode::Client => self.client_conn.complete_io(&mut self.client_buf),
-            Mode::Server => self.server_conn.complete_io(&mut self.server_buf),
-        } {
-            Ok(_) => Ok(()),
-            Err(err) => {
-                if err.kind() == std::io::ErrorKind::WouldBlock {
-                    Ok(())
-                } else {
-                    Err(err)
-                }
-            }
+    /// Treat `WouldBlock` as an `Ok` value
```

Comment:
Can we include why it should be treated an an `Ok` value?

Reply:
Added minimally more ("for when blocking is expected"). I don't want to explain here the cases where blocking would be expected, since I feel like that's more appropriate to comment where `ignore_block` is actually used (in the handshake and in `recv()`). Even so, it's self-explanatory there too: it makes sense that a connection would block in the middle of a handshake with our synchronous blocking local buffer IO.